ui/po/pt_BR.po

```diff
@@ -387,9 +387,9 @@ msgid ""
 msgstr ""
 " Se você instalou via AUR, certifique-se de ter executado o comando de pós-"
 "instalação recomendado:\n"
-" systemctl --user enable --now xr-driver.service\n"
+"          systemctl --user enable --now xr-driver.service\n"
```

Comment:
Did you mean to add this spacing?

Reply:
I inserted them just because I saw these same spaces in the original English sentences. Aren't them needed?

Reply:
Oh, the english-language line was getting cut off in the diff, so I didn't see that. I don't think it's necessary but it's fine to match the english version. Thanks!